### PR TITLE
#20 - add speakers list

### LINKED ISSUE #20: Speakers list
I would like to have nice and tidy public view for speakers list.

### PR DESCRIPTION
**Pull request by JakubZakowicz**
I have created simple views for speakers list

- Speakers list
speakers_list

- Add or Edit Speaker
add_speaker

This should close #20

### CODE REVIEW

**Review by kamilpiech97 — DISMISSED** [review]

**Review by EwelinaLasowy — APPROVED** [review]

**Review by MrHDOLEK — DISMISSED** [review]

File: resources/views/speakers/create.blade.php [hunk 1]
```diff
@@ -1,42 +1,92 @@
...
+                                    class="shadow-sm focus:ring-indigo-500 focus:border-indigo-500 block w-full sm:text-sm border-gray-300 rounded-md">{{ old('description') }}</textarea>
+                                <x-input-error for="description" />
+                            </div>
+                        </div>
+                        <div>
+                            <label for="avatar" class="block font-medium text-gray-700">
+                                Avatar
+                            </label>
+                            <div x-data class="flex items-center gap-12 mt-4">
+                                <img x-ref="image" id="image" src="{{ asset('/static/images/no_avatar.png') }}"
```
Comment (mtracz) [inline resources/views/speakers/create.blade.php:41]: Could you convert `png` to `webp`? `no_avatar.png`

File: resources/views/speakers/edit.blade.php [hunk 2]
```diff
@@ -1,44 +1,94 @@
...
-        <div>
-            <h1>Edit Speaker</h1>
-            @auth
-                <form method="post" action="{{ route('speakers.update', $speaker) }}" enctype="multipart/form-data">
-                    @method('PUT')
-                    @csrf
+    <div class="container md:w-[800px] mx-auto">
+        @auth
+            <form method="post" action="{{ route('speakers.update', $speaker) }}" enctype="multipart/form-data">
+                @method('PUT')
```
Comment (mtracz) [inline resources/views/speakers/edit.blade.php:7]: Which method is correct? POST or PUT?

**Review by mtracz — CHANGES_REQUESTED** [review]

File: resources/views/speakers/edit.blade.php (hunk 2 above)
Comment (krzysztofrewak) [reply resources/views/speakers/edit.blade.php:7]: This is a stupid Laravel feature: https://laravel.com/docs/9.x/blade#method-field

Comment (mtracz) [reply resources/views/speakers/edit.blade.php:7]: Ok, got it.

**Review by mtracz — APPROVED** [review]

**Pull request merged**